common/src/main/resources/bootstrap.yml

```diff
@@ -0,0 +1,14 @@
+spring:
+  application:
+    name: config-task
+  profiles:
+    active: development
+  cloud:
+    config:
+      uri: http://localhost:8888
+      username: jxjxjxmxnxs
```

Comment:
remember avoid to use usernames and passwords in configuration files ... use things like `${USER_NAME}` so it can be taken from the environment